backend/app/api/endpoints/internal/devices.py

```
@@ -0,0 +1,69 @@
+        description="Optional device capability required for routing",
+    )
+    device_id: str | None = Field(
+        default=None,
+        description="Optional explicit device ID override",
+    )
+
+
+@router.post("/sandbox/exec", response_model=DeviceSandboxExecResponse)
+async def execute_device_sandbox_command_internal(
+    request: InternalDeviceSandboxExecRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)